internal/db/schema/migrations/oss/postgres/99/02_worker_controller_tables.up.sql

```diff
@@ -40,6 +40,20 @@ create table server_worker (
   create_time wt_timestamp,
   update_time wt_timestamp,
   version wt_version,
+  last_status_time timestamp with time zone,
+  constraint last_status_time_after_create_time
```

Comment:
Should this maybe be `last_status_time_not_before_create_time` since the check is >=?